.travis.yml

```diff
@@ -2,14 +2,15 @@ language: go
 go:
 - 1.5.1
 - tip
+before_install:
+  - rvm install 2.2.3
+  - nvm install 5.5
```

Comment:
Alrighty, so these are now being explicitly upgraded rather than using the Travis-CI defaults.